<?xml version="1.0"?>

<launch>
  <master auto="start"/>

  <!-- NESTOR robot -->
  <node pkg="nestor_command" type="nestor_voice" respawn="true" name="NESTOR_voice" output="screen">
  </node>

  <!-- NESTOR robot -->
  <node pkg="nestor_command" type="nestor_weather" respawn="false" name="NESTOR_weather" output="screen">
  </node>
  
  <!-- NESTOR robot -->
  <node pkg="nestor_command" type="nestor_radio" respawn="false" name="NESTOR_radio" output="screen">
  </node>
  
  <!-- NESTOR robot -->
  <node pkg="nestor_command" type="nestor_calendar" respawn="true" name="NESTOR_calendar" output="screen">
  </node>

  <!-- NESTOR robot -->
  <node pkg="nestor_command" type="command_manager" respawn="false" name="NESTOR_command_manager" output="screen">
  </node>

  <!-- NESTOR robot -->
  <node pkg="nestor_command" type="light_manager" respawn="true" name="NESTOR_light_manager" output="screen">
  </node>

  <!-- NESTOR robot -->
  <node pkg="nestor_command" type="heat_manager" respawn="true" name="NESTOR_showerheat_manager" output="screen">
  </node>

</launch>
